blomsterLandet/src/App.js

```diff
@@ -12,9 +13,33 @@ import ShopScreen from './screens/ShopScreen';
 const TabNavigator = createBottomTabNavigator({
     Shop: ShopScreen,
     Plants: HomeScreen,
-    Notification: NotificationScreen,
```

Comment:
The word "Notification" is used in src/services/NotifService.js
It would also have to be changed on that location.